File: jdroid-java-webapp-core/src/main/java/com/jdroid/javaweb/sentry/SentryLoggerFactory.java
package com.jdroid.javaweb.sentry;

import org.slf4j.ILoggerFactory;
import org.slf4j.Logger;
import org.slf4j.LoggerFactory;

public class SentryLoggerFactory implements ILoggerFactory {

	@Override
	public Logger getLogger(String name) {
		return new SentryLogger(LoggerFactory.getLogger(name));
	}
}

File: jdroid-java-webapp-core/src/main/java/com/jdroid/javaweb/sentry/SentryLoggerFactory.kt
package com.jdroid.javaweb.sentry

import org.slf4j.ILoggerFactory
import org.slf4j.Logger
import org.slf4j.LoggerFactory

class SentryLoggerFactory : ILoggerFactory {

    override fun getLogger(name: String): Logger {
        return SentryLogger(LoggerFactory.getLogger(name))
    }
}
